### PR TITLE
Fixed documentation for convertArea

### PR DESCRIPTION
**Pull request by dwaggenspack**
The convertArea documentation listed some units in their singular form.  The function uses the units in plural form.

Please fill in this template.

- [ x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x ] Have read [How To Contribute](https://github.com/Turfjs/turf/blob/master/CONTRIBUTING.md#how-to-contribute).
- [x ] Run `npm test` at the sub modules where changes have occurred.
- [ x] Run `npm run lint` to ensure code style at the turf module level.

Submitting a new TurfJS Module.

- [ ] Overview description of proposed module.
- [ ] Include JSDocs with a basic example.
- [ ] Execute `./scripts/generate-readmes` to create `README.md`.
- [ ] Add yourself to **contributors** in `package.json` using "Full Name <@GitHub Username>".

### CODE REVIEW

**Comment by DenisCarriere** [comment]
Thanks @dwaggenspack for your contributions, however you need to edit the Javascript files instead of the README's, these are considered read-only since they are auto-generated from the source code of the Javascript files.

You'll find your edit in the following line:
https://github.com/Turfjs/turf/blob/master/packages/turf-helpers/index.js#L525

**Comment by dwaggenspack** [comment]
I am still a little new to git.  how should I go about redoing this?

**Comment by DenisCarriere** [comment]
You don't need to revert anything, you just need to simply add that same documentation comment to the `index.js` file

**Comment by rgdonohue** [comment]
Thank you for the help @DenisCarriere. We're running through the library and want to know how best help make it better.

**Comment by DenisCarriere** [comment]
@rgdonohue 👍 Great! Definitely here to help out with any PR's.

The easiest way to help is to include more documentation in the `index.js` files, after that's done you can run the [scripts/generate-readmes](https://github.com/Turfjs/turf/blob/master/scripts/generate-readmes) to re-populate the README's.

Once that's done, we publish those updates to the Turf website.

https://github.com/Turfjs/turf-www

It's a bit of a process, but once you do a few commits it's not too bad.

File: packages/turf-helpers/index.js [hunk 1]
```diff
@@ -522,7 +522,7 @@ export function convertLength(length, originalUnit, finalUnit) {
 
 /**
  * Converts a area to the requested unit.
- * Valid units: kilometers, kilometres, meters, metres, centimetres, millimeter, acre, mile, yard, foot, inch
+ * Valid units: kilometers, kilometres, meters, metres, centimetres, millimeters, acres, miles, yards, feet, inches
```
Comment (DenisCarriere) [inline packages/turf-helpers/index.js]: @dwaggenspack I've added your doc comment to the `index.js` source code

**Pull request merged**